net/tstun/linkattrs_notlinux.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !linux
+// +build !linux
+
+package tstun
+
+func setLinkAttrs(iface tun.Device) error {
```

Comment:
This "tun." reference without an import fails all the non-Linux CI builders.

Reply:
Oops, my bad :/